src/components/App/App.module.css

```diff
@@ -1,21 +1,85 @@
+body {
+  background-image: url('./Media/wall4.jpg');
+  min-height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+}
+
 .wrap {
-  display: grid;
+  display: flex;
+  flex-flow: column;
   margin: auto;
-  width: 600px;
+  max-height: 90%;
   padding: 50px;
-  grid-template-areas: "sidebar ... content content";
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  font-size: 1 rem;
+  background-color: white;
+  width: 100%;
+  padding-left: 10px; 
+}
+
+.icon {
+  height: 50px;
+  width: auto;
+}
+
+.menu {
+  width: 650px;
+  display: flex;
+  justify-content: center;
+  margin: auto;
 }
 
 .link {
   text-decoration: none;
-  font-family: 'Courier New', Courier, monospace;
-  color: black;
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+  margin: 20px;
+  color: #fff;
+  width: 11.6rem;
+  padding: .77rem 0;
+  border-radius: 60px;
+  background-color: #001E00;
```

Comment:
Codacy found an issue: [Expected "#001E00" to be "#001e00" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)